### DISCUSSION TITLE
Add more hash for content comparisons: CRC32 sha1 sha256

### DISCUSSION THREAD

**Discussion by beng-tang** [hsoft/dupeguru#1152] [Ideas]
Hi just a suggestion to add options to use different checksum methods to check file contents:
CRC32, sha1, and sha256, in addition to MD5.

**Comment by arsenetar** [comment]
We no longer use md5, and use xxhash for file hashing (there is fallback to md5 but all official builds will end up using xxhash).  I don't see a strong reason to support multiple hashing algorithms as it adds complexity without really bringing much benefit.
